nowcasting_dataset/filesystem/utils.py

```diff
@@ -97,6 +97,8 @@ def check_path_exists(path: Union[str, Path]):
 
     `path` can include wildcards.
     """
+    if not bool(path):
```

Comment:
TODO: Don't need the bool

Reply:
add issue? actually i dont really understand `bool(path)`